.npmignore

```diff
@@ -0,0 +1,12 @@
+node_modules/
+.idea/
+docs/
+website/
+
+# This is a library
```

Comment:
https://pics.onsizzle.com/im-here-for-the-hookers-and-the-booze-sir-this-3277512.png

Reply:
Yes, I am laughing about my own jokes.